opm/flowdiagnostics/utility/AssembledConnections.cpp

```diff
@@ -256,9 +256,7 @@ CSR::accumulateRowEntries(const int               numRows,
     }
 
     // Note index range: 1..numRows inclusive.
-    for (decltype(this->ia_.size())
-             i = 1, numRows = this->ia_.size() - 1;
-         i <= numRows; ++i)
+    for (int i = 1; i <= numRows; ++i)
```

Comment:
> Do you suggest that whenever using `std::vector<>::operator[]()` we should use only the `size_type`?

Actually, yes.  Mostly because the API is specified in terms of `size_type`.  Rewriting statements that were originally written to conform to the API requirements to use a type that does not meet the requirement is heading in the wrong direction.